### ISSUE TITLE
ShellUTest fails

### ISSUE THREAD

**Issue by vsbogd** [opencog/cogserver#28]
ShellUTest probably after formatting changes in `opencog/atomspace`.

One of the cases:
```guile> 
guile> <<<
/ws/cogserver/build/tests/shell/../../../tests/shell/ShellUTest.cxxtest:76: Error: Expected (274 <= reso.size()), found (274 > 265)
Error: Create rep 29 of 30 in thread 11 failed
	Got: 265 >>>opencog> Entering scheme shell; use ^D or a single . on a line by itself to exit.
guile> (EvaluationLink
  (PredicateNode "visible face")
  (ListLink
    (NumberNode "21039")))
```

Complete test log:
[shellutest.20200615.log](https://github.com/opencog/cogserver/files/4780499/shellutest.20200615.log)

**Comment by linas** [comment]
Fixed in pull req #29 .. thanks for noticing this... I run the tests manually, but apparently, not often enough.

**Comment by linas** [comment]
BTW, the size change was this: remove 5 trailing blank spaces, remove 2 newlines, remove 2 leading blank spaces: new message is 5+2+2=9 bytes shorter than the old one.
